client/blocks/reader-post-card/conversation-post.jsx

```diff
@@ -20,7 +20,16 @@ class ConversationPost extends React.Component {
 	};
 
 	render() {
-		const commentIdsToShow = map( take( this.props.comments, 3 ), 'ID' );
+		const commentIdsToShow = map(
+			take(
+				filter(
+					this.props.comments,
+					comment => comment.type !== 'trackback' && comment.type !== 'pingback'
+				),
+				3
```

Comment:
I know you didn't change it here ...but can we move this magic number to a var? `numberOfCommentsToShow`?